ui/expensedetail/src/main/java/com/kappzzang/jeongsan/expensedetail/ExpenseDetailActivity.kt

```diff
@@ -48,4 +55,21 @@ class ExpenseDetailActivity : AppCompatActivity() {
         binding.expenseDetailItemListRecyclerview.layoutManager =
             LinearLayoutManager(this, LinearLayoutManager.VERTICAL, false)
     }
+
+    private fun initiateViewModel() {
+        getExpenseId()
+    }
+
+    private fun getExpenseId() {
+        val expenseId = intent?.getParcelableData<String>(
+            ExpenseDetailContract.EXPENSE_ID
+        )
+
+        expenseId?.let {
+            viewModel.updateExpenseIdAndInit(it)
+        } ?: let {
+            Toast.makeText(this, "지출 정보를 불러오는 데 실패했습니다.", Toast.LENGTH_LONG).show()
+            viewModel.updateExpenseIdAndInit("0")
+        }
```

Comment:
P3: expenseId 가 null일 경우 0을 지정해서 특정 값을 보내기 보다는 nullable로 보내서 후에 판단하는건 어떨까요?!
```suggestion
    if (expenseId == null) {
        Toast.makeText(this, "지출 정보를 불러오는 데 실패했습니다.", Toast.LENGTH_LONG).show()
    }   
    viewModel.updateExpenseIdAndInit(expenseId)
}
```

Reply:
해당 코드는 지출 정보를 전달하는 로직이 구현이 안 되있어 무조건 실패하는 분기입니다.
더미 데이터 표시를 위해 임시로 코드를 작성해놓았고, 구현이 끝나면 expenseId가 null인 경우 이전 페이지로 돌아가는 등 추가적인 예외 처리를 해야할 것 같습니다.
아직 API 통합이 덜 끝나서 수정 예정인 부분인데, 주석 처리를 안해서 혼란을 빚은것 같네요.